### DISCUSSION TITLE
Is it possible to have an multiple areas on screen?

### DISCUSSION THREAD

**Discussion by prnvbn** [pterm/pterm#615] [Q&A] [answered]
I want a way to show two areas next to each other. Is this possible somehow? I tried to use sections but that doesn't work as I expected it to. Any help will be greatly appreciated :)

```go
	var done = make(chan bool)
	pterm.DefaultSection.Println("This is a section!")

	area, _ := pterm.DefaultArea.WithCenter().Start()
	area.Update("aaaaaaaaaaaaaaaaaaaaaaaaaa")
	go func() {
		defer area.Stop()
		for i := 0; i < 1; i++ {
			area.Update("bbbbbbbbbbbbbbbbbbbbbbbbbb")
		}
		done <- true
	}()

	pterm.DefaultSection.WithLevel(1).Println("This is another section!")
```

**Comment by MarvinJWendt** [comment, marked as answer]
Hi, this feature is not directly supported by PTerm, but you could use the `PanelPrinter` to print two panels next to each other. You can use the output of the `PanelPrinter` (`Srender()`) and pipe that into an `AreaPrinter`. Let me know if that works for you!

**Reply by prnvbn** [reply]
That worked well, but I ended up using a table :)
